src/runtime/zarf.yaml

```diff
@@ -0,0 +1,28 @@
+kind: ZarfPackageConfig
+metadata:
+  name: uds-core-runtime
+  description: "UDS Core Runtime"
+  url: "https://github.com/defenseunicorns/uds-runtime"
+
+components:
+  - name: uds-runtime
```

Comment:
Could/Should we be importing this from the published runtime package?

Reply:
Unfortunately that's only possible with skeleton I believe - and I'm not sure runtime is publishing/should be publishing a skeleton package? I think that pattern is generally being discouraged?

Reply:
I think you can only import remote components as skeleton packages, which we don't currently publish

Reply:
I should refresh more lol. Just deleted my comment saying ^^. I was also under the impression 💀 packages are discouraged